mmeds/config.py

```diff
@@ -284,6 +311,8 @@
 MIXS_MAP = {v: k for (k, v) in MMEDS_MAP.items()}
 
 
+
```

Comment:
What are these extra white space lines for?

Reply:
Getting the get_salt function out of the way. I removed them.